//! `false` fallback.

use std::process::exit;

fn main() {
    exit(1)
}